cdp_scrapers/instances/seattle-static.json

```diff
@@ -0,0 +1,166 @@
+{
+    "persons": {
+        "Alex Pedersen": {
+            "name": "Alex Pedersen",
+            "is_active": true,
+            "router_string": null,
+            "email": null,
+            "phone": null,
+            "website": null,
+            "picture_uri": "http://www.seattle.gov//assets/images/Council/Members/Pedersen/Councilmember-Alex-Pedersen_homepage-banner.jpg",
+            "seat": {
+                "name": "Position 4",
+                "electoral_area": "District 4",
+                "electoral_type": null,
+                "image_uri": "https://uploads.visitseattle.org/2015/03/Suzzallo_Reading_Room_NicholasBoos-720x313.jpg",
+                "external_source_id": null,
+                "roles": null
+            },
+            "external_source_id": null
+        },
+        "Andrew Lewis": {
+            "name": "Andrew Lewis",
+            "is_active": true,
+            "router_string": null,
+            "email": null,
+            "phone": null,
+            "website": null,
+            "picture_uri": "http://www.seattle.gov//assets/images/Council/Members/Lewis/lewis_homepage_banner.jpg",
+            "seat": {
+                "name": "Position 7",
+                "electoral_area": "District 7",
+                "electoral_type": null,
+                "image_uri": "https://uploads.visitseattle.org/2015/03/QueenAnne_LempelZiv-720x313.jpg",
+                "external_source_id": null,
+                "roles": null
+            },
+            "external_source_id": null
```

Comment:
Can you change all the electoral area's to "Central District", "Wallingford", "Ballard", etc.

Reply:
Ah, got it. I misunderstood the proper meaning of that field.

Reply:
Sorry, have follow-up question. [This page](https://www.seattle.gov/neighborhoods/neighborhoods-and-districts) lists "geographic areas" for the districts. Do you like these? Or go with more specific neighborhood names like you suggested above?

Reply:
Yea that's perfect! For the Position 8 and 9 I would use "Citywide"